deepchecks/recommender/suites.py

```diff
@@ -14,27 +14,16 @@
 Each function returns a new suite that is initialized with a list of checks and default conditions.
 It is possible to customize these suites by editing the checks and conditions inside it after the suites' creation.
 """
-from typing import Callable, Dict, List, Union
-
 from deepchecks.recommender.checks import (LabelPopularityDrift, OperationsAmountSegmentPerformance, PopularityBias,
                                            PredictionPopularityDrift, SamplePerformance, ScatterPerformance)
 from deepchecks.tabular import Suite
-from deepchecks.tabular.checks import (BoostingOverfit, CalibrationScore, ConflictingLabels, ConfusionMatrixReport,
-                                       DataDuplicates, DatasetsSizeComparison, DateTrainTestLeakageDuplicates,
-                                       DateTrainTestLeakageOverlap, FeatureDrift, FeatureFeatureCorrelation,
-                                       FeatureLabelCorrelation, FeatureLabelCorrelationChange,
-                                       IdentifierLabelCorrelation, IndexTrainTestLeakage, IsSingleValue, LabelDrift,
-                                       MixedDataTypes, MixedNulls, ModelInferenceTime, MultivariateDrift,
-                                       NewCategoryTrainTest, NewLabelTrainTest, OutlierSampleDetection, PercentOfNulls,
-                                       PredictionDrift, RegressionErrorDistribution, RocReport, SimpleModelComparison,
-                                       SingleDatasetPerformance, SpecialCharacters, StringLengthOutOfBounds,
-                                       StringMismatch, StringMismatchComparison, TrainTestPerformance,
-                                       TrainTestSamplesMix, UnusedFeatures, WeakSegmentsPerformance)
```

Comment:
Why not all tabular checks? Except the specific model evaluation ones such as CalibrationScore or BoostingOverfit

Reply:
I copied from the prod suite

Reply:
if they want integrity let them run it alone